### PR TITLE
add option to enable fedramp

### PR DESCRIPTION
**Pull request by taymoork2**
[SPARK-312429](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-312429)

image

### CODE REVIEW

File: demo/App.jsx [hunk 1]
```diff
@@ -56,19 +61,35 @@ export default function App() {
...
+                alignItems: 'BaseLine',
+              }}
+            >
+              <Button color="blue" type="submit" onClick={handleSaveToken} disabled={!!token} ariaLabel="Save Token">
+                Save Token
+              </Button>
+              <Button type="button" onClick={handleClearToken} disabled={!token} ariaLabel="Clear Token">
+                Clear Token
+              </Button>
+              <Checkbox
```
Comment (lalli-flores) [inline demo/App.jsx:76]: Not sure it makes sense to just add a random checkbox next to the buttons. You should create a separate `FedRAMP` section (before the `Access Token` section) that explains where to get a FedRAMP token and the checkbox to enable the environment

File: demo/App.jsx [hunk 2]
```diff
@@ -56,19 +61,35 @@ export default function App() {
               type="password"
               value={tokenInput}
             />
-            <Button color="blue" type="submit" onClick={handleSaveToken} disabled={!!token} ariaLabel="Save Token">
-              Save Token
-            </Button>
-            <Button type="button" onClick={handleClearToken} disabled={!token} ariaLabel="Clear Token">
-              Clear Token
-            </Button>
+            <span
```
Comment (lalli-flores) [inline demo/App.jsx:64]: What's this?

**Comment by lalli-flores** [comment]
You are missing a section in the Meeting widget README that explains how to enable the widget for FedRAMP. Similar to the [SDK's](https://github.com/webex/webex-js-sdk#fedramp-environment)

File: demo/App.jsx [hunk 3]
```diff
@@ -41,11 +46,19 @@ export default function App() {
           <h5>Webex Widgets require an access token to identify the current user.</h5>
           <h5>
             You can get an access token from{' '}
-            <a href="https://developer.webex.com" target="_blank">
-              developer.webex.com
+            <a href={`https://developer${fedramp ? '-usgov' : ''}.webex.com`} target="_blank">
+              {`developer${fedramp ? '-usgov' : ''}.webex.com`}
```
Comment (taymoork2) [inline demo/App.jsx:50]: only cause browser to manipulate strings, not rerender entire elements

**Review by lalli-flores — APPROVED** [review]

**Pull request merged**